### ISSUE TITLE
Testing instructions missing from Readme

### ISSUE THREAD

**Issue by ltfschoen** [paritytech/parity-ethereum#10655]
The Readme does not include instructions on how to run tests for an individual package and how to run tests across all the packages at once like in Substrate https://github.com/paritytech/substrate#613-shared-steps

**Comment by ltfschoen** [comment]
Suggest adding in the "Documentation" section after https://github.com/paritytech/parity-ethereum/pull/10652 is merged, since users may want to specify a specific package to run the tests on, and we've listed them in that section

**Comment by ltfschoen** [comment]
Note that these are the commands that I commonly use to quickly view docs (including private doc comments) and to run tests whilst outputting logs at the same time.

```
cargo doc -p evmbin --no-deps --document-private-items --open
cargo test -p evmbin -- --nocapture
```

**Comment by niklasad1** [comment]
Closed by #10835
